src/style.css

```diff
@@ -0,0 +1,26 @@
+body {
+  background-color: #f1f2f5;
+}
+
+.card {
+  background-color: white;
+  width: 40%;
+  margin: 100px auto 0 auto;
+  box-shadow: 0 2px 8px rgba(0, 0, 0, 0.2);
+}
+
+#todo-list {
+  color: black;
+  list-style: none;
+  padding: 0;
+}
+
+p,
+ul {
+  margin: 0;
+}
+
+.list-item {
+  padding: 15px;
+  border-bottom: 1px solid rgba(98, 57, 57, 0.07);
```

Comment:
- [ ]  I would recommend keeping the same wireframe design as you have given. For now, it's not looking the same. Please work on that. Ofcourse, you don't have to implement the functionality for now but make sure the design remains the same. :slightly_smiling_face: 

### Your design

![Screenshot from 2021-09-02 03-43-06](https://user-images.githubusercontent.com/22434660/131752403-cf33d95a-8d04-43da-a9c9-5e6015d7fe3e.png)


### Given design

![Screenshot from 2021-09-02 03-42-54](https://user-images.githubusercontent.com/22434660/131752411-7738bbc1-3cb2-43f4-aa36-387d35977f50.png)